.github/workflows/poll-starter-workflows.yml

```diff
@@ -0,0 +1,80 @@
+name: Poll Starter Workflows
+
+on:
+  schedule:
+    - cron: "0 */6 * * *"
+  workflow_dispatch:
+
+env:
+  STARTER_WORKFLOW_PATH: "./resources/yaml/"
+  STARTER_WORKFLOW_REPO_BRANCH: "partner_templates"
```

Comment:
💡 Just for archive reasons: What was the reason that there is a difference between `main` branch and `partner_templates` branch?  (I know that's a repo level question but I really want to avoid multiple vscode level commits which will clutter the `main` branch commit history against this repo.) Overhead of too many PRs.

Reply:
partner_templates is where we are asked to make PRs against.